SignalMessaging/contacts/OWSContactsManager.m

```diff
@@ -258,12 +380,13 @@ - (void)intersectContactsWithRetryDelay:(double)retryDelaySeconds
         // TODO: Abort if another contact intersection succeeds in the meantime.
```

Comment:
The exponential backoff logic.  

Jeff as requested that we ensure that there's never more than one CDS "contact sync/request batch" in flight at a time.  De-bouncing will be non-trivial though.

Reply:
NSOperation/OWSOperation with a non-concurrent may be helpful in that future work.